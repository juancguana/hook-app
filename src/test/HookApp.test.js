import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp';

describe('Test in <HookApp />', () => {
  test('should show correctly', () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
