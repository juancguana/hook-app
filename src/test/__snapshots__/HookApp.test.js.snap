// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <HookApp /> should show correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
